src/Illuminate/Bus/Queueable.php

```diff
@@ -74,14 +88,45 @@ public function delay($delay)
     /**
      * Set the jobs that should run if this job is successful.
      *
-     * @param  array  $chain
+     * @param      $chain
+     * @param null $queue
```

Comment:
This doc is not correct I'm afraid. This says that you can only pass `null` through.

Reply:
done

Reply:
@GrahamCampbell - can you approve this change review please.